packages/react-reconciler/src/ReactFiberReconciler.new.js

```diff
@@ -323,9 +323,9 @@ export function updateContainer(
   parentComponent: ?React$Component<any, any>,
   callback: ?Function,
 ): Lane {
-  if (__DEV__) {
```

Comment:
This is not needed. It's helpful to see from a glance that the call is DEV-only regardless of what's in the function.